examples/reference/widgets/ChatEntry.ipynb

```diff
@@ -75,8 +77,76 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pn.widgets.ChatEntry(\n",
-    "    value=pn.pane.Markdown(\"## Happy you're here!\"), user=\"Happy User\", avatar=\"😀\"\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "data = np.random.rand(5, 2)\n",
+    "dataframe = pd.DataFrame(data, columns=list(\"AB\"))\n",
+    "\n",
+    "fig = plt.figure(figsize=(5,2.5))\n",
+    "ax = fig.add_subplot(111)\n",
+    "ax.plot(data)\n",
+    "plt.close(fig)\n",
```

Comment:
I prefer less unrelated code for reference gallery; perhaps something like (haven't tested):
```python
import matplotlib.pyplot as plt

figs, axes = plt.subplots(1, 1)
axes[0].plot([1, 2, 3])
```
would suffice to demo support for Matplotlib?